index.html

```diff
@@ -59,14 +59,23 @@ <h1>
             <div id="sortbutton">
               <button id="sortBtn" onclick="sortRecipes()">AtoZ</button>
             </div>
+           <div class="linebr"></div>
+            <div id="stopWatch">
+              <h1>Cooking Timer</h1>
+              <input id="cookingTime" type="number" placeholder="Insert Your Time in minuits" value= "0" required>
+              <button onclick="startCookingTimer()">start</button>
```

Comment:
Prefer using `addEventListener` instead of `onclick` attributes

Reply:
thank you for review, will do that

Reply:
it's done now